docs/data-management/post-session.md

```diff
@@ -286,8 +286,19 @@ To support backward compatibility (and some extra, currently unsupported feature
     ```
 
 ### Convert eye-tracking into BIDS with *bidsphysio*
+- [ ] Pull the latest docker image of esavary/bidsphysio with:
+``` shell
+docker pull esavary/bidsphysio
+```
+- [ ] Open [schedule_ET_data.tsv](../code/physioconv/schedule_ET_data.tsv) to find the phase encoding direction and the name of the .EDF file corresponding to the session you want to convert.
+- [ ] In the folder containing your data, create a new folder named metadata. Copy the file containing information about the eye-tracker, named [info_ET.json](../code/physioconv/info_ET.json), inside this metadata folder.
+- [ ]  Run the following command for the file corresponding to the DWI (Diffusion-Weighted Imaging) data, replacing PATH_RAW_DATA,PATH_OUTPUT, DWI_FILENAME, SESSION_NUMBER, and PHASE_ENCODING_DIRECTION with the information corresponding to the file you want to process:
```

Comment:
Please use real names (examples) rather than placeholders.
```suggestion
- [ ]  Run the following command for the file corresponding to the DWI (Diffusion-Weighted Imaging) data, replacing PATH_RAW_DATA,PATH_OUTPUT, DWI_FILENAME, SESSION_NUMBER, and PHASE_ENCODING_DIRECTION with the information corresponding to the file you want to process:
```

Reply:
I think it can make it more confusing as to what exactly needs to be changed and where when running the command, especially when directly including the session number and phase encoding direction. Would it not be better to include this in the notebook or as an example?

Reply:
I don't think the notebook is a replacement for this. I'm happy also to have this example in the notebook, but I believe this is easier to understand with specific examples.

If it were only one field, then I'd probably be okay with it, but the number of parameters to replace is a mouthful and they only extend the example code (e.g., `PHASE_ENCODING_DIRECTION` instead of just `LR`).

Reply:
As you prefer. I've updated it with a real example.

Reply:
Thx!